addons/csi-azurefile/ccm-azurefile.yaml

```diff
@@ -762,15 +739,17 @@ metadata:
     app.kubernetes.io/component: csi-driver
     app.kubernetes.io/part-of: azurefile-csi-driver
     app.kubernetes.io/managed-by: Helm
-    app.kubernetes.io/version: "v1.29.2"
-    helm.sh/chart: azurefile-csi-driver-v1.29.2
+    app.kubernetes.io/version: "v1.30.2"
+    helm.sh/chart: azurefile-csi-driver-v1.30.2
   annotations:
-    csiDriver: "v1.29.2"
-    snapshot: "v6.3.1"
+    csiDriver: "v1.30.2"
+    snapshot: "v7.0.2"
 spec:
   attachRequired: false
   podInfoOnMount: true
   volumeLifecycleModes:
     - Persistent
     - Ephemeral
   fsGroupPolicy: ReadWriteOnceWithFSType
+  tokenRequests:
+    - audience: api://AzureADTokenExchange
```

Comment:
Do we have any idea what is this exactly doing?

Reply:
https://blog.identitydigest.com/azuread-federate-k8s/ says

> The best practice for these tokens is to have a unique audience value for each provider you intend to use the token. The webhook uses the Microsoft recommended value for tokens meant for Azure AD (api://AzureADTokenExchange). This value needs to match the value you configured the Federated Identity Credential on your Azure AD application.

The official docs @ https://learn.microsoft.com/en-us/entra/workload-id/workload-identity-federation-create-trust?pivots=identity-wif-apps-methods-azp#important-considerations-and-restrictions say

> audiences lists the audiences that can appear in the external token. Required. You must add a single audience value, which has a limit of 600 characters. The recommended value is "api://AzureADTokenExchange". It says what Microsoft identity platform must accept in the aud claim in the incoming token.